src/server/enrichment/validation/index.js

```diff
@@ -102,6 +103,8 @@ const bodyHandler = body =>  {
  */
 const errorHandler = ( error, query ) => {
   logger.error( error );
+  // something fishy - blow out the cache
+  pcCache.reset();
```

Comment:
Pretty sure that would delete everything.  It would be better to delete only the specified key, or even better to leverage the promise chain.